### PR TITLE
Remove deprecated decorators and loaders

### PR DESCRIPTION
**Pull request by jonspalmer**
## What I did

Removed a bunch of legacy deprecated code

### CODE REVIEW

File: addons/a11y/src/preset/addDecorator.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import { withA11Y } from '../index';
+import { withA11y } from '../index';
 
-export const decorators = [withA11Y];
+export const decorators = [withA11y];
```
Comment (jonspalmer) [inline addons/a11y/src/preset/addDecorator.ts]: made me giggle that the preset was using the deprecated version...

File: MIGRATION.md [hunk 2]
```diff
@@ -750,6 +763,25 @@ Furthermore, the decorator `checkA11y` has been deprecated and renamed to `withA
 
 See the [a11y addon README](https://github.com/storybookjs/storybook/blob/master/addons/a11y/README.md) for more information.
 
+### Addon centered decorator deprecated
```
Comment (ndelangen) [inline MIGRATION.md]: I added a parameter called `layout` and it and 1 of it's possible values is `center`.

This automatically centers the component.

It also supports `fullscreen` and `padded`.

I think we could drop addon-centered entirely in 6.0.0.

Comment (shilman) [reply MIGRATION.md]: Yes, let's please move it to the deprecated addons repo, pretty please 🙏  

We can merge this first and follow up separately: https://github.com/storybookjs/storybook/issues/10118

**Review by shilman — APPROVED** [review]
LGTM! ❤️

**Pull request merged**